src/validator.js

```diff
@@ -337,12 +337,19 @@ export class Validator {
         const errors = []
         if (Array.isArray(schema.required)) {
           schema.required.forEach(e => {
+            if (this._checkType('string', value[e]) && value[e].length === 0) {
+              errors.push({
+                path: `${path}.${e}`,
+                property: 'required',
+                message: this.translate('error_required', [e])
+              })
+            }
             if (typeof value[e] !== 'undefined') return
             const editor = this.jsoneditor.getEditor(`${path}.${e}`)
             /* Ignore required error if editor is of type "button" or "info" */
             if (editor && ['button', 'info'].includes(editor.schema.format || editor.schema.type)) return
             errors.push({
-              path,
+              path: `${path}.${e}`,
```

Comment:
this line allows to generate the errors inside the child fields

Reply:
Are you sure that this is not `breaking-change`?

Reply:
If it is, this should be based on an `option` with default value that keeps the current behaviour

Reply:
Let's make this optional and disabled by default